end_to_end/signal_handler_test.go

```diff
@@ -40,9 +69,12 @@ var _ = Describe("Signal handler tests", func() {
 				rng := rand.New(rand.NewSource(time.Now().UnixNano()))
 				time.Sleep(time.Duration(rng.Intn(1000)+500) * time.Millisecond)
 				_ = cmd.Process.Signal(unix.SIGINT)
+
+				getOutput(outScanner, &output)
```

Comment:
how is this goroutine synchronised with main thread? Are we sure that this output read completely before runing expects in main thread below?